### PR TITLE
DM-47906 fix to pass processing_type to different jobs

### PR DESCRIPTION
**Pull request by zhaoyuyoung**
## Checklist

- [ ] ran Jenkins
- [ ] added a release note for user-visible changes to `doc/changes`

### CODE REVIEW

File: doc/changes/DM-47906.fix.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+fix to pass processing_type to different jobs
```
Comment (MichelleGower) [inline doc/changes/DM-47906.fix.rst:1]: Commit messages should be present tense.  Towncrier (doc/changes) should be past tense.  So "Fixed the passing of processing_type..."

Comment (MichelleGower) [reply doc/changes/DM-47906.fix.rst:1]: Also file should be named DM-47906.bugfix.rst

**Review by MichelleGower — APPROVED** [review]
Comments about doc/changes file.  Merge approved.

**Review by wguanicedew — COMMENTED** [review]
It's good.

**Pull request merged**